.github/workflows/backport.yml

```
@@ -0,0 +1,20 @@
+name: Backport
+on:
+  pull_request:
+    types: [closed]
+jobs:
+  backport:
+    name: Create backport PRs
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.3.4
```

We're using `@v2` in the other actions.

```suggestion
      - uses: actions/checkout@v2
```

---

Shouldn't we update the other actions then?

---

Why do they need to be pinned at `2.3.4`?

---

Didn't know about that, thanks!